erdjs/src/esdtHelpers.ts

```diff
@@ -0,0 +1,53 @@
+import {ScArgumentsParser} from "./scArgumentsParser";
+import {ErrInvalidEsdtTransferDataField} from "./errors";
+import BigNumber from "bignumber.js";
+
+/**
+ * This class exposes static methods that are useful for parsing ESDT transfer transactions
+ */
+export class EsdtHelpers {
+
+    /**
+     * This function will return the token identifier and the amount from a given data field for an ESDT transfer, or
+     * an exception if something went wrong
+     * @param dataField dataField represents the data filed to extract esdt transfer data from
+     */
+    public static extractFieldsFromEsdtTransferDataField(dataField: string) {
+        if (!dataField.includes("ESDTTransfer", 0)) {
```

Comment:
```suggestion
        if (!dataField.startsWith("ESDTTransfer@")) {
```
Someone may call a method which only contains `ESDTTransfer` in the name (eg. `myESDTTransferMethod`), so it's best to guard against any prefix / suffix.

Reply:
true. done